common/api/core-frontend.api.md

```diff
@@ -1251,7 +1251,7 @@ export function areaToEyeHeightFromGcs(view3d: ViewState3d, area: GlobalLocation
 
 // @internal
 export interface AttachToViewportArgs {
-    drawingToSheetTransform?: Transform;
+    // (undocumented)

```

Comment:
What happened here? This is documented on master.

Reply:
Actually, almost none of these api.md file changes should be present.

Reply:
I pulled the branch and re-ran extract-api - seems to be OK now.

Reply:
It's some shenanigans when I merged in from master and then did an extract-api. If I can't fix it on this branch, I'll just start over.